### DISCUSSION TITLE
my repository isn't showing that it's detected the language I'm working with; python

### DISCUSSION THREAD

**Discussion by Selyiah** [community/community#68947] [New to GitHub] [answered]
Hi everyone, 

I'm new to Github and my Python repository isn't showing the language being used, I'd like to know how I can add this so that it says Python language. I've added the workflow and added a file to detect false/true. 

Not sure what else to do? please help, thanks

**Comment by radar-blip** [comment]
Your python file needs an extension (filename.py)
Also, there shouldn't be a space in the filename.

**Reply by satilpereira** [reply, marked as answer]
Exactly, your file [password gen](https://github.com/Selyiah/python/blob/main/password%20gen) should be named to something like password-gen.py or so.

**Comment by Selyiah** [comment]
Thank you all so much!

**Comment by Abkeyev99** [comment]
Click "Marked as answer" Thanks so good :)
